aws/resource_aws_rds_cluster.go

```diff
@@ -455,23 +477,19 @@ func resourceAwsRDSClusterCreate(d *schema.ResourceData, meta interface{}) error
 			opts.Port = aws.Int64(int64(attr.(int)))
 		}
 
-		if attr, ok := d.GetOk("enabled_cloudwatch_logs_exports"); ok && len(attr.([]interface{})) > 0 {
-			opts.EnableCloudwatchLogsExports = expandStringList(attr.([]interface{}))
+		if attr, ok := d.GetOk("preferred_backup_window"); ok {
+			modifyDbClusterInput.PreferredBackupWindow = aws.String(attr.(string))
+			requiresModifyDbCluster = true
 		}
 
-		// Check if any of the parameters that require a cluster modification after creation are set
-		var clusterUpdate bool
-		if attr := d.Get("vpc_security_group_ids").(*schema.Set); attr.Len() > 0 {
-			clusterUpdate = true
-			opts.VpcSecurityGroupIds = expandStringList(attr.List())
+		if attr, ok := d.GetOk("preferred_maintenance_window"); ok {
+			modifyDbClusterInput.PreferredMaintenanceWindow = aws.String(attr.(string))
+			requiresModifyDbCluster = true
 		}
 
-		if attr, ok := d.GetOk("db_cluster_parameter_group_name"); ok {
-			opts.DBClusterParameterGroupName = aws.String(attr.(string))
-		}
-
-		if _, ok := d.GetOk("backup_retention_period"); ok {
-			clusterUpdate = true
+		if attr := d.Get("vpc_security_group_ids").(*schema.Set); attr.Len() > 0 {
+			opts.VpcSecurityGroupIds = expandStringList(attr.List())
+			// requiresModifyDbCluster = true
```

Comment:
is this intentionally commented out?

Reply:
Great catch! For some reason the old code had it (so it was being set both during create and calling an update) and I was experimenting 🔬 -- experiment was a success! I'll remove it. 👍